uaa/src/test/java/org/cloudfoundry/identity/uaa/degraded/DegradedSamlLoginTests.java

```diff
@@ -101,17 +122,15 @@ public void testGetTokenKey() throws Exception {
         RestTemplate restTemplate = new RestTemplate();
         MultiValueMap<String, String> headers = new LinkedMultiValueMap<>();
         headers.add("Accept", APPLICATION_JSON_VALUE);
-        headers.set("Authorization", getAuthorizationHeader("app", "appclientsecret"));
+        headers.set("Authorization", getAuthorizationHeader(basicAuthClientId, basicAuthClientSecret));
         HttpEntity getHeaders = new HttpEntity(headers);
         ResponseEntity<Map> tokenKeyGet = restTemplate.exchange(
-                "http://localhost:8080/uaa" + "/token_key",
+                protocol + baseUaaZoneHost + "/token_key",
                 HttpMethod.GET,
                 getHeaders,
                 Map.class
         );
         assertEquals(HttpStatus.OK, tokenKeyGet.getStatusCode());
-        assertEquals("MAC", tokenKeyGet.getBody().get("kty"));
-        assertEquals("HS256", tokenKeyGet.getBody().get("alg"));
     }
```

Comment:
These are distinct for local uaa and ge uaa. 
The tokenKey is successfully retrieved, these were additional asserts.